index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <link rel="stylesheet" href="styles.css" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <section class="container">
+      <nav>
+        <header class="toolbar">
+          <div class="logo">
+            <div class="placeholder">
+              <h3>My Logo</h3>
+            </div>
+          </div>
+
+          <div class="normal-button">
+            <div id="icon-menu">
+              <div class="menu-line"></div>
+              <div class="menu-line"></div>
+              <div class="menu-line"></div>
+            </div>
+          </div>
```

Comment:
- [ ] [REQUIRED] Please ensure that the hamburger icon has a `nav` tag. This will help us have a semantic element that gives greater accessibility.
-  [ ] [OPTIONAL ] For the hamburger icon, you can get an icon from [font-awesome icons](https://fontawesome.com/icons) and use its SVG or image directly.